tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "module": "CommonJS",
```

Comment:
it allows us for imports in ts code.

You can notice that in our application we use babel to compile ts files, and ts is used only for typing hints.
However server.ts is not babelified.